examples/next/pages/ui/components/account-settings/change-password/index.page.tsx

```diff
@@ -0,0 +1,34 @@
+import { Amplify } from 'aws-amplify';
+
+import {
+  Authenticator,
+  ChangePassword,
+  Button,
+  Card,
+} from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+import awsExports from './aws-exports';
+Amplify.configure(awsExports);
+
+export default function AuthenticatorWithEmail() {
+  return (
+    <>
```

Comment:
This fragment is extraneous i think?

Reply:
Oh shoot, I did not mean to add this yet. Please disregard [🤦](https://emojipedia.org/person-facepalming/)